src/lib/utils.ts

```diff
@@ -26,6 +26,11 @@ export function pick<T, K extends keyof T>(obj: T, keys: K[]): Pick<T, K> {
 
 export const isServerSide = () => typeof window === "undefined"
 
+export const isViewTransition = () =>
+  // @ts-ignore
+  !!document.startViewTransition &&
```

Comment:
This function will throw an error on the server side, so we need to use isServerSide to exclude server-side execution.
<img width="614" alt="image" src="https://github.com/Crossbell-Box/xLog/assets/8266075/7c047b26-d897-4a62-a2e9-bad4ad05759c">

Reply:
@DIYgod Thanks ❤️ !  And I added `stack context` to make the animation look more natural,